src/server/terrain/structure_building_blocks.zig

```diff
@@ -133,8 +141,57 @@ pub const StructureBuildingBlock = struct {
 		for(childBlockStringId.items, 0..) |colorName, colorIndex| {
 			self.children[colorIndex] = try initChildTableFromZon(stringId, colorName, colorIndex, childrenZon.getChild(colorName));
 		}
+		self.updateBlueprintChildLists();
 		return self;
 	}
+	pub fn updateBlueprintChildLists(self: StructureBuildingBlock) void {
+		for(self.children, 0..) |child, index| {
+			if(child.items.len == 0) continue;
+
+			var found = false;
+			for(self.blueprints[0].childBlocks) |blueprintChild| {
+				if(blueprintChild.index != index) continue;
+				found = true;
+				break;
+			}
+			if(found) break;
```

Comment:
Wait, so it just ignores errors for all children that are past the first found one?

Also please use labelled break/continue instead of bools